cluster_utils/base/__init__.py

```diff
@@ -0,0 +1 @@
+"""Base modules of cluster_utils that are shared between the client and server."""
```

Comment:
Should we add a comment here noting that external dependencies in this package should be kept minimal?

Reply:
done